index.js

```diff
@@ -0,0 +1,302 @@
+//   let a = prompt('choise -3,2,5, 3 or 0')
+//   let b = (a > 0 && a < 5) ? "Верно" : "Неверно"
+//   alert(b);
+// }
+// logicFunctional();
+
+
+// function logicFunctionalSecond() {
+//   let a = prompt('choise -3,2,5 or 0')
+//   a = Number(a);
+//   let b = (a == 0 || a == 2) ? a + 2 : a / 10
+//   alert(b);
+// }
+// logicFunctionalSecond();
+
+
+
+// function logicFunctionalThird() {
+//   let a = prompt('choise 1,0,3 ')
+//   let b = prompt('choise 3,6,5 ')
+
+//   a = Number(a);
+//   b = Number(b);
+//   let c = (a <= 1 && b >= 3) ? a + b : a - b
+//   alert(c);
+// }
+// logicFunctionalFourth();
+
+// function logicFunctionalFourth() {
+//   let a = prompt('input number')
+//   let b = prompt('input number ')
+
+//   a = Number(a);
+//   b = Number(b);
+//   let c = ((a > 2 && a < 11) || (b >= 6 && b < 14)) ? "True" : "False"
+//   alert(c);
+// }
+// logicFunctionalFourth();
+
+
+
+//For, while
+
+// function countFor() {
+//   let i = 1;
+//   for (1 == 1; i <= 50; i++)
+//     document.writeln(i);
+// }
+
+// countFor()
+
+// function countwhile() {
+//   let i = 1;
+//   while (i < 51) {
+//     document.writeln(i++);
+//   }
+// }
+// countwhile()
+
+// function countwhileThird() {
+//   let i = 1;
+//   let a = 0;
+//   while (i < 101) {
+//     if (i % 2 == 0) {
+
+//       a = a + i
+//       document.writeln(`четное число ${i + '<br>'}`);
+//       document.writeln(`сумма ${a + '<br>'}`);
+//     }
+//     i++
+//   }
+// }
+// countwhileThird()
+// function countInteration() {
+//   let i = 0;
+//   let n = 1000;
+//   while (n > 50) {
+//     n = n / 2;
+//     let k = n;
+
+//     i++;
+
+//   }
+//   // if (n < 50) n = n * 2;
+
+//   document.writeln(`последнее число ${n + '<br>'}`);
+
+//   document.writeln(`количество интераций ${i + '<br>'}`);
+// }
+// countInteration()
+
+
+
+// //switc;
+
+// let num = prompt(" Введите число от 1 до 4")
+// let result;
+// num = Number(num);
+// switch (num) {
+//   case 1:
+//     result = "зима";
+//     break;
+//   case 2:
+//     result = "весна";
+//     break;
+//   case 3:
+//     result = "лето";
+//     break;
+//   case 4:
+//     result = "осень";
+//     break;
+//   default:
+//     result = "Вы ввели не то число";
+// }
+// document.writeln(result);
+
+//  В переменной day лежит какое-то число из интервала от 1 до 31. Определите в какую декаду месяца попадает это число (в первую, вторую или третью).
+
+// function DecadeOfTheMoth() {
+//   let day = prompt("Введите число месяца");
+//   day = Number(day);
+//   if (day >= 1 && day < 8) {
+//     alert("Первая Декада");
+//   }
+//   else if (day >= 8 && day < 16) {
+//     alert("Первая Декада");
+//   }
+//   else if (day >= 16 && day < 23) {
+//     alert("Третья Декада");
+//   }
+//   else if (day >= 23 && day <= 30) {
+//     alert("Четверая Декада");
+//   }
+//   else {
+//     alert("Недопустимое значение")
+//   }
+
+// }
+// DecadeOfTheMoth()
+
+
+// function DecadeOfTheYear() {
```

Comment:
functions should be named in camelCase - the first latter is always small

and function names should start from verb

for example:

getDecadeOfTheYear

handleClick

getMonth

calculateSum